mypy.ini

```diff
@@ -1,2 +1,15 @@
 [mypy]
+show_error_codes = True
+exclude = (?x)(
+          setup\.py
+;          | tests/
```

Comment:
Was this commented line meant to be left in here?

Reply:
I think I subconsciously left it in as a discussion point.  I added the mypy.ini changes because they were handy and didn't affect the way pre-commit hooks scan the code.  To be consistent we should probably ignore the tests directly here as well.  However, I commented it out to double check any possible errors in my newly added test code.

The unfortunate problem with doing that was mypy doesn't always catch problems in test functions that have fixture arguments that are _not_ type hinted.  So it's a bit of a wash at the moment.  My gut says we should either completely ignore the tests or fully commit to ensuring the tests are typed, linted, etc.